daemon/SecretsImpl/secretsrequestprocessor.cpp

```diff
@@ -343,8 +343,10 @@ Daemon::ApiImpl::RequestProcessor::createCustomLockCollection(
     ikdRequest.setOperation(InteractionParameters::CreateCollection);
     ikdRequest.setInputType(InteractionParameters::AlphaNumericInput);
     ikdRequest.setEchoMode(InteractionParameters::PasswordEchoOnEdit);
-    ikdRequest.setPromptText(tr("Enter the passphrase which will be used to encrypt the new collection %1 in plugin %2")
-                             .arg(collectionName, storagePluginName));
+    //: This will be displayed to the user, prompting them to enter a passphrase which will be used to encrypt a collection.  %1 is the application name, %2 is the collection name, %3 is the plugin name.
+    //% "Application %1 wants to create a new secrets collection %2 in plugin %3.  Enter the passphrase which will be used to encrypt the collection."
```

Comment:
UI lingo is "apps", not "applications". If app icon is shown in the future I would drop the whole "App" prefix.

Reply:
Ok, changed to "App %1 ..."